Block/Form/Moto.php

```diff
@@ -0,0 +1,295 @@
+use Adyen\Payment\Helper\Data;
+use Adyen\Payment\Helper\Installments;
+use Adyen\Payment\Helper\Recurring;
+use Adyen\Payment\Helper\Vault;
+use Adyen\Payment\Logger\AdyenLogger;
+use Magento\Backend\Model\Session\Quote;
+use Magento\Customer\Model\Session;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\View\Element\Template\Context;
+
+class Moto extends \Magento\Payment\Block\Form\Cc
+{
+    /**
+     * @var string
+     */
+    protected $_template = 'Adyen_Payment::form/moto.phtml';
+
+    /**
+     * @var Data
+     */
+    protected $adyenHelper;
+
+    /**
+     * @var \Magento\Framework\App\State
+     */
+    protected $appState;
+
+    /**
+     * @var \Magento\Checkout\Model\Session
+     */
+    protected $checkoutSession;
+
+    /**
+     * @var Installments
+     */
+    private $installmentsHelper;
+
+    /**
+     * @var Installments
+     */
+    private $chargedCurrency;
+
+    /**
+     * @var Quote
+     */
+    private $backendCheckoutSession;
+
+    /**
+     * @var AdyenLogger
+     */
+    private $adyenLogger;
+
+    /**
+     * @var Config
+     */
+    private $configHelper;
+
+    /**
+     * @var Session
+     */
+    private $customerSession;
+
+    /**
+     * @var Vault
+     */
+    private $vaultHelper;
+
+    /**
+     * Cc constructor.
+     *
+     * @param Context $context
+     * @param \Magento\Payment\Model\Config $paymentConfig
+     * @param Data $adyenHelper
+     * @param \Magento\Checkout\Model\Session $checkoutSession
+     * @param Quote $backendCheckoutSession
+     * @param Installments $installmentsHelper
+     * @param ChargedCurrency $chargedCurrency
+     * @param AdyenLogger $adyenLogger
+     * @param Config $configHelper
+     * @param Session $customerSession
+     * @param Vault $vaultHelper
+     */
+    public function __construct(
+        Context $context,
+        \Magento\Payment\Model\Config $paymentConfig,
+        Data $adyenHelper,
+        \Magento\Checkout\Model\Session $checkoutSession,
+        Quote $backendCheckoutSession,
+        Installments $installmentsHelper,
+        ChargedCurrency $chargedCurrency,
+        AdyenLogger $adyenLogger,
+        Config $configHelper,
+        Session $customerSession,
+        Vault $vaultHelper
+    ) {
+        parent::__construct($context, $paymentConfig);
+        $this->adyenHelper = $adyenHelper;
+        $this->appState = $context->getAppState();
+        $this->checkoutSession = $checkoutSession;
+        $this->backendCheckoutSession = $backendCheckoutSession;
+        $this->installmentsHelper = $installmentsHelper;
+        $this->chargedCurrency = $chargedCurrency;
+        $this->adyenLogger = $adyenLogger;
+        $this->configHelper = $configHelper;
+        $this->customerSession = $customerSession;
+        $this->vaultHelper = $vaultHelper;
+    }
+
+    /**
+     * @return string
+     * @throws \Adyen\AdyenException
+     */
+    public function getClientKey()
```

Comment:
Remove this method since `clientKey` is being fetched from the data attribute of the dropdown in the frontend.